test/users_finish_test.rb

```diff
@@ -153,6 +162,12 @@
 
             subject.run
           end
+
+          it "does not write the auto-login config" do
+            expect(Yast::Autologin).to_not receive(:Write)
```

Comment:
I wonder whether the correct thing to do is to not call to `Write`. Maybe we should actually call `User(false) && Write`to ensure it's disabled?

Reply:
Good point!